tests/vmi_multus_test.go

```diff
@@ -580,6 +580,36 @@ var _ = Describe("SRIOV", func() {
 
 	tests.BeforeAll(func() {
 		tests.BeforeTestCleanup()
+		// Check if the hardware supports SRIOV
+		nodes := tests.GetAllSchedulableNodes(virtClient)
+		Expect(nodes.Items).ToNot(BeEmpty(), "There should be some compute node")
+
+		for _, node := range nodes.Items {
+			var sriovallocate bool = false
+			var sriovcapacity bool = false
+			resourceList := node.Status.Allocatable
+			for k, v := range resourceList {
+				if strings.HasPrefix(string(k), sriovResourceName) {
```

Comment:
I think we are looking for perfect match. If the `sriovResourceName` is set for `example.com/net`, but the cluster is configured only with `example.com/net1`, this check would pass but test would fail anyway.

Reply:
Done